<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WTraffic</class>
 <widget class="QWidget" name="WTraffic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>650</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuery">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>650</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>152</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbDeviceId">
        <property name="text">
         <string>节点名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combDeviceId"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>间隔时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combDisTime">
        <item>
         <property name="text">
          <string>5分钟</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15分钟</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30分钟</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1小时</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1天</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1个月</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbTime">
        <property name="text">
         <string>查询时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="deStartTime"/>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="deEndTime"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="graphWid" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>681</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>320</y>
     <width>631</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnLeftBig">
      <property name="text">
       <string>《</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLeft">
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUp">
      <property name="text">
       <string>放大</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDown">
      <property name="text">
       <string>缩小</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPageDown">
      <property name="text">
       <string>上页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPageUp">
      <property name="text">
       <string>下页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRight">
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRightBig">
      <property name="text">
       <string>》</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
